### DISCUSSION TITLE
Orion Grid Connect Smart Wi-Fi Entrance

### DISCUSSION THREAD

**Discussion by noinformationavailable** [make-all/tuya-local#812] [General]
I saw that you have support for Orion locks. I have the model shown https://www.bunnings.com.au/orion-grid-connect-smart-wi-fi-entrance-lock_p0330511 and have not been able to connect to it.

It is set up and present in tuya IOT cloud and I have both the device id and local key

I have tried to connect using the ip address assigned to it (dhcp and mac address assigned)

Is there something more specific I should be looking at with it?

**Comment by make-all** [comment]
This is a battery powered device.  It probably cannot be reliably connected to, as it will be offline most of the time to save batteries.  Battery powered devices only work via the cloud.

**Comment by phatrickk** [comment]
What is the orion lock that functions locally with tuya local? I can't find seem to find it online. Anyone know?

**Reply by make-all** [reply]
[Orion DL021HA](https://grid-connect.com.au/smart-home-products/door-hardware-locks/Orion%20DL021HA) is supported, and works through a Bluetooth hub, so doesn't have the issues that a WiFi lock would have with being battery powered.  There is another device that was added based on config info, but I am not sure if it is working, or ended up having the battery issues (it was around then that I put devices without logs on lowest priority for implementation to avoid wasting time on devices which have no chance of working).
